src/main/java/games/strategy/triplea/ui/UndoablePlacementsPanel.java

```diff
@@ -11,10 +10,6 @@ public UndoablePlacementsPanel(final GameData data, final AbstractMovePanel move
     super(data, movePanel);
   }
 
-  protected final String getSpecificComponentForMoveLabel(final Route route) {
```

Comment:
This method had no callers, nor could I find any possible references via reflection.